src/Pages/Coins/index.js

```diff
@@ -1,11 +1,14 @@
 import React, { Component } from 'react'
+import CryptoChart from '../../Components/CryptoChart'
```

Comment:
you don't have the absolute imports setup?

Reply:
components not Components
You will have problems because of the naming
https://stackoverflow.com/questions/10523849/changing-capitalization-of-filenames-in-git